doc/user/content/third-party/supported-tools.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Supported Tools Overview"
+description: "Get details about third-party tool support with Materialize"
+menu:
+  main:
+    parent: 'third-party'
+---
+
+Materialize fully supports the following third-party tools:
+
+| Tool | Purpose |
+|------|---------|
+| [Docker](docker.md) | Easily deploy Materialize and other required infrastructure.
+| [Debezium](debezium.md) | Propagate change data capture (CDC) data from an upstream database to Materialize.
```

Comment:
We may want to be selective about our support for Debezium, since it also depends on the source database. We  don't support Debezium + Postgres, for example.

Reply:
I thought we did support PG––isn't that what symbiosis mode uses? asking just for my own edification

Reply:
I also don't know what's missing from our Debezium + Postgres support! But I'm happy to add the info.